app/Repositories/ContactRepositoryTimer.php

```diff
@@ -0,0 +1,33 @@
+<?php namespace App\Repositories;
+
+use App\Models\Contact;
+use Illuminate\Contracts\Pagination\LengthAwarePaginator;
+
+class ContactRepositoryTimer implements ContactRepository
+{
+    private $repo;
+
+    public function __construct(ContactRepository $contact_repository)
+    {
+        $this->repo = $contact_repository;
+    }
+    
+    public function getAll($nbrPages, $parameters): LengthAwarePaginator
+    {
+        $start_time = microtime(true);
+        $contacts = $this->repo->getAll($nbrPages, $parameters);
+        $end_time = microtime(true);
+
+        $time = $end_time - $start_time;
+
+        $class = get_class($this->repo);
```

Comment:
Output the full class that is being timed, makes it easy to compare times later on.